docs/content/authentication.md

```diff
@@ -23,15 +23,13 @@ See Grafana's [Authentication API
 docs](https://grafana.com/docs/grafana/latest/http_api/auth/) for more info.
 
 ## Grafana Cloud Prometheus
-To interact with Grafana Cloud Prometheus, you must have the `cortextool` binary
-available on your path (download it [here](https://github.com/grafana/cortex-tools/releases)),
-and must have these environment variables set:
+To interact with Grafana Cloud Prometheus, you must have these environment variables set:
 
 | Name | Description | Required |
 | --- | --- | --- |
 | `CORTEX_ADDRESS` | URL for Grafana Cloud Prometheus instance | true |
-| `CORTEX_TENANT_ID` | Tenant ID for your Grafana Cloud Prometheus account | true |
-| `CORTEX_API_KEY` | Authentication token/api key | true |
+| `CORTEX_TENANT_ID` | Tenant ID for your Grafana Cloud Prometheus account | false |
+| `CORTEX_API_KEY` | Authentication token/api key | false |
```

Comment:
surely these are required to interact with Grafana Cloud Prometheus? Perhaps not for Cortex or Mimir alone though.

Reply:
Indeed, just thought if we are targeting standalone Cortex here as well, as mentioned below: 

> Note, this will also work with other Cortex installations, alongside Grafana Cloud Prometheus.

Not critical of course, maybe I should revert that.

Reply:
That is fine, but the words need to be valid: Grafana Cloud Prometheus requires creds, Cortex/Mimir don't. So long as it is clear, happy to have it in the docs.

Reply:
Decided to revert for now, not sure how to put it right, without making this page overcomplicated.